src/utils/errors.ts

```diff
@@ -78,6 +79,10 @@ export const errorHandler = (err: Error, req: Request, res: Response, next: Next
             res.status(400);
             break;
         default:
+            logger.error("Internal Server Error");
+            logger.error(err.name);
+            logger.error(err.message);
+            logger.error(err.stack);
```

Comment:
These four messages could be separated by logs from other requests happening at same time, even another error causing debug confusion... perhaps you want these 4 messages combined into one?